scripts/cgp_data_loader.py

```diff
@@ -46,7 +46,7 @@ def main(argv=sys.argv[1:]):
     input_format = subparsers.add_parser("standard", help='Standard CGP DSS input file format')
     input_format.add_argument("--json-input-file", metavar="JSON_INPUT_FILE", required=True,
                               help="Path to the standard JSON format input file")
-    input_format.add_argument('-c', '--concurrently', action='store_true', default=False,
+    input_format.add_argument('--linear', action='store_false', default=True,
```

Comment:
Maybe `--serial` or `--sequential` instead of `--linear`?

Also, the help string needs to be updated.

Reply:
Whoops! Ok that's much better. I was struggling to come up with a better name.